finances/index.html

```diff
@@ -17,8 +17,15 @@ <h3>by fiscal year (June 1 - May 31)</h3>
       <h3 class=top-spacer><b>Foundation Activities</b></h3>
       <p>The okTurtles Foundation funds a number of activities in support of beneficial decentralization technologies.</p>
 
+      <p class="top-spacer"><b>FY2023</b></p>
+      <p>The Foundation's fundraising activities for FY2023 <a href=/donate>are ongoing</a>.</p>
+
       <p class="top-spacer"><b>FY2022</b></p>
-      <p>The Foundation's fundraising activities for FY2022 <a href=/donate>are ongoing</a>.</p>
+      <p>The okTurtles Foundation received $41,757.91, 2.73233405 BTC, 322.15946856 XMR, and 240.8511658 ZEC in new donations during FY2022. During FY2022 the okTurtles Foundation paid out $1,377.20, 4.04971016 BTC and 220.817579881 XMR to support its activities.</p>
+      <ul>
+        <li>The foundation continued to fund development on the <a href="https://groupincome.org">Group Income Prototype</a></li>
```

Comment:
Can you add another bullet point linking to Shelter Protocol?

Reply:
I didn't include it because Shelter Protocol was not released until FY23, just like the Group Income prototype. We could add a list under the current year, while leaving the numbers "in progress" if you'd like.

Reply:
That's definitely correct. I'll think on wording.

Reply:
No special wording is needed because a) it's already been released and b) that's phrasing things the same as we have for the Group Income Prototype.